sentry-native-sample/src/main/java/io/sentry/nativesample/MainActivity.java

```diff
@@ -9,11 +11,26 @@
 
   @Override
   protected void onCreate(Bundle savedInstanceState) {
+    districtMode();
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_main);
 
+    Timber.plant(new Timber.DebugTree());
+
     Timber.i("Sentry.isEnabled() = %s", Sentry.isEnabled());
 
     findViewById(R.id.native_capture).setOnClickListener(view -> NativeSample.message());
   }
+
+  private void districtMode() {
```

Comment:
districtMode -> strictMode?

Reply:
yep, typo.
missed this PR too -> https://github.com/getsentry/sentry-android/pull/258